src/SFA.DAS.FAT.Web/Views/Shared/_RenderSelectedFilter.cshtml

```diff
@@ -0,0 +1,8 @@
+﻿@model Tuple<string, string, string>
```

Comment:
Would not the list partial work for single items, instead of this? Or only have partial that renders one clear filter item and then use that in a loop where needed?